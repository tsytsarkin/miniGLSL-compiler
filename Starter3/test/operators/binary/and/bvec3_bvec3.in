{ bvec3 a; bvec3 b; c = a && b; }
