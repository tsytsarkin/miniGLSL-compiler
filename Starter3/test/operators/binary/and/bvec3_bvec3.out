  1: { bvec3 a; bvec3 b; c = a && b; }
 (SCOPE (DECLARATIONS (DECLARATION a bvec3) (DECLARATION b bvec3)) (STATEMENTS (ASSIGN unknown c (BINARY bvec3 && a b))))
